content/en/docs/concepts/services-networking/service.md

```diff
@@ -187,6 +187,13 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Over Capacity Endpoints
+Starting in Kubernetes 1.21, if an Endpoints resource has more than 1000
+endpoints the controlplane will add an annotation
+`endpoints.kubernetes.io/over-capacity: warning` and value `warning` to indicate
+that the Endpoints resource is over capacity. In future releases, addressess
+will will be truncated after the first 1000.
```

Comment:
We actually [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in concept documentation (with a few documented exceptions).

Reply:
Removed the statement about the future functionality.